e2e/cypress/integration/api/nodemanager/node_manager_type.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { uuidv4 } from '../../../support/helpers';
```

Comment:
:1st_place_medal:

Reply:
all your helper functions/the way you wrote the tests made it SO easy to write this one up, thank you!!